src/Services/DocumentTemplates/DocumentTemplateRender.php

```diff
@@ -109,6 +109,7 @@ public function doRender($template_id, $template_content = null, $json_data = nu
         // purify html (and remove js)
         $isLegacy = stripos($template, 'portal_version') === false;
         $config = HTMLPurifier_Config::createDefault();
+        $config->set('Cache.SerializerPath', $GLOBALS['temporary_files_dir']);
```

Comment:
Do we need a subfolder in the temp directory for this?  Probably overly paranoid but just wondering if there's a risk  of other files in the apache php processor space being overwritten (session files, upload files).  It'd be a supply chain attack on htmlpurifier, so probably too paranoid here.

Reply:
wondered about that but we need a writable folder somewhere

Reply:
opened #7520 to address this but maybe you meant a separate dir in `/tmp` not in `sites/default/documents/temp` then the system purges it instead of having it hang around in documents

Reply:
I was meaning a separate dir in /tmp.  Depends on how much we care about shared hosts.  Share hosts sometimes lock down the /tmp directory and writing to the configured OpenEMR temp directory would probably be the least surprising for someone whose configured that global option.  Not sure if HTMLPurifier cleans up its own cache files though.